apps/app/src/views/routes/Settings/components/users/UsersList.tsx

```diff
@@ -54,10 +43,9 @@ const usersQuery = graphql(/* GraphQL */ `
 
 export const UsersList = (props: { rolesMap: Record<string, string> }) => {
   const { clinicalClient } = usePhoton();
-  const [sortBy, setSortBy] = useState<'NAME' | 'ROLES' | 'EMAIL' | undefined>();
-  const [sortByDir, setSortByDir] = useState(false);
+  // const [sortBy, setSortBy] = useState<'NAME' | 'ROLES' | 'EMAIL' | undefined>();
```

Comment:
Leaving these in here for when we put sorting back

Reply:
can you make a ticket to re-add ?